eslzArm/managementGroupTemplates/policyDefinitions/Databricks/DENY-Databricks-Sku.json

```diff
@@ -0,0 +1,57 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-08-01/managementGroupDeploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {},
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.Authorization/policyDefinitions",
+            "apiVersion": "2020-03-01",
+            "name": "Deny-Databricks-Sku",
+            "properties": {
+                "policyType": "Custom",
+                "mode": "Indexed",
+                "displayName": "Deny non-premium databricks sku",
+                "description": "Enfoces the use of Premium Databricks workspaces.",
+                "parameters": {
+                    "effect": {
+                        "type": "String",
+                        "metadata": {
+                            "displayName": "Effect",
+                            "description": "Enable or disable the execution of the policy"
+                        },
+                        "allowedValues": [
+                            "Audit",
+                            "Disabled",
+                            "Deny"
+                        ],
+                        "defaultValue": "Deny"
+                    }
+                },
+                "policyRule": {
+                    "if": {
+                        "allOf": [
+                            {
+                                "field": "type",
+                                "equals": "Microsoft.Databricks/workspaces"
+                            },
+                            {
+                                "field": "Microsoft.DataBricks/workspaces/sku.name",
+                                "notEquals": "premium"
+                            }
+                        ]
+                    },
+                    "then": {
+                        "effect": "[[parameters('effect')]"
+                    }
+                },
+                "metadata": {
+                    "version": "1.0.0",
+                    "category": "Databricks",
+                    "preview": false,
+                    "deprecated": false
```

Comment:
lets drop "preview" and "deprecated" in the metadata object

Reply:
DONE